src/EditorFeatures/CSharpTest/Diagnostics/UseImplicitOrExplicitType/UseImplicitTypeTests.cs

```diff
@@ -1443,13 +1443,41 @@ static void M()
                 diagnosticSeverity: DiagnosticSeverity.Error);
         }
 
+        private static string trivial2uple =
+                    @"
+namespace System
+{
+    public class ValueTuple
+    {
+        public static ValueTuple<T1, T2> Create<T1, T2>(T1 item1, T2 item2) => new ValueTuple<T1, T2>(item1, item2);
+    }
+    public struct ValueTuple<T1, T2>
+    {
+        public T1 Item1;
+        public T2 Item2;
+
+        public ValueTuple(T1 item1, T2 item2) { }
+    }
+} ";
+
         [WpfFact, Trait(Traits.Feature, Traits.Features.CodeActionsUseImplicitType), Trait(Traits.Feature, Test.Utilities.Traits.Features.Tuples)]
         public async Task SuggestVarOnLocalWithIntrinsicTypeTuple()
         {
+            var before = @"class C { static void M() { [|(int a, string)|] s = (a: 1, ""hello""); } }";
+            var after = @"class C { static void M() { var s = (a: 1, ""hello""); } }";
+
+            await TestAsync(before, after, options: ImplicitTypeEverywhere(), parseOptions: TestOptions.Regular.WithTuplesFeature(), withScriptOption: true);
+
+            await TestMissingAsync(before, options: ImplicitTypeWhereApparent(), parseOptions: TestOptions.Regular.WithTuplesFeature(), withScriptOption: true);
```

Comment:
Ah, TestMissingAsync, I see. In that case, adding the workitem would seem helpful because then we have context about why it started failing when someone fixes this.